test/parallel/test-domain-uncaught-exception.js

```diff
@@ -88,13 +88,13 @@ function test4() {
   const d5 = domain.create();
   const d6 = domain.create();
 
-  d5.on('error', function onErrorInD2Domain(err) {
+  d5.on('error', function onErrorInD2Domain() {
     process.send('errorHandledByDomain');
   });
 
   d5.run(function() {
     d6.run(function() {
-      setTimeout(function onTimeout() {
+      setImmediate(function onSetImmediate() {
```

Comment:
Ditto. We're testing that domains and `setTimeout()` interact the way they are supposed to, so it needs to be `setTimeout()`. Add a duration of `1` rather than changing it to `setImmedaite()`.